eventsourced/src/lib.rs

```diff
@@ -76,7 +83,7 @@ pub trait EventSourced {
     const TYPE_NAME: &'static str;
 
     /// The event handler.
-    fn handle_evt(self, evt: &Self::Evt) -> Self;
+    fn handle_evt(self, evt: Self::Evt) -> Self;
```

Comment:
I now also changed this definition again to enable `handle_evt` to consume the event.